### DISCUSSION TITLE
Why is password encryption not supported with SQLite?

### DISCUSSION THREAD

**Discussion by DotCat1985** [OrchardCMS/OrchardCore#9020] [Q&A]
Hi,
If I select SQLite database on setup page, OrchardCore connects to it using the following string connection:

`Data Source=yessql.db;Cache=Shared`

So, the CMS doesn't allow to use connect to SQLite db using a connection string with password like this:

`Data Source=yessql.db;Cache=Shared;Password=password;`

So, why is it not supported in OrchardCore if I use SQLite database?

**Comment by hishamco** [comment]
If I'm not wrong the credentials will be encrypted in the config file using secrets. @deanmarcussen correct me if I'm wrong

**Comment by deanmarcussen** [comment]
SQLite files are stored on disc.
The assumption is that anyone with access to the disc, has access to the db file.

I don't believe SQLite directly supports encrypting a db, i.e putting a password on the file.

https://stackoverflow.com/questions/5669905/sqlite-with-encryption-password-protection
